From e96ee4c45b4a3bd0f8aa1e6602ea05f0165365f1 Mon Sep 17 00:00:00 2001
From: Ben Wijen <ben@wijen.net>
Date: Tue, 9 Dec 2025 19:23:45 +0100
Subject: [PATCH] align SigThreadProc #ifdef

The other SigThreadProc #ifdef only check for WIN32
Make sure compilation on cygwin (and msys2) doesn't break
---
 src/mosquitto_broker_internal.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/mosquitto_broker_internal.h b/src/mosquitto_broker_internal.h
index 7516f312..1d9832d6 100644
--- a/src/mosquitto_broker_internal.h
+++ b/src/mosquitto_broker_internal.h
@@ -847,8 +847,10 @@ void service_install(void);
 void service_uninstall(void);
 void service_run(void);
 
+#ifdef WIN32
 DWORD WINAPI SigThreadProc(void* data);
 #endif
+#endif
 
 /* ============================================================
  * Websockets related functions
-- 
2.49.0.windows.1

